[2022-08-16 15:25:49,798] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:25:40.438085+00:00 [queued]>
[2022-08-16 15:25:49,850] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:25:40.438085+00:00 [queued]>
[2022-08-16 15:25:49,852] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 15:25:49,852] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-16 15:25:49,853] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 15:25:49,916] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2022-08-16 15:25:40.438085+00:00
[2022-08-16 15:25:49,929] {standard_task_runner.py:52} INFO - Started process 6986 to run task
[2022-08-16 15:25:49,948] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2022-08-16T15:25:40.438085+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag.py', '--cfg-path', '/tmp/tmpmz3_rkft', '--error-file', '/tmp/tmp7ch9f2w9']
[2022-08-16 15:25:49,957] {standard_task_runner.py:80} INFO - Job 43: Subtask insert_data
[2022-08-16 15:25:50,336] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:25:40.438085+00:00 [running]> on host b232b3e4166c
[2022-08-16 15:25:50,784] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-16T15:25:40.438085+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-16T15:25:40.438085+00:00
[2022-08-16 15:25:50,843] {logging_mixin.py:115} INFO - rate_date:  2022-08-16
[2022-08-16 15:25:50,846] {logging_mixin.py:115} INFO - value_:  24207.92857100000037
[2022-08-16 15:25:50,878] {base.py:68} INFO - Using connection ID 'postgre_test_db' for task execution.
[2022-08-16 15:25:50,928] {python.py:173} INFO - Done. Returned value was: None
[2022-08-16 15:25:50,973] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20220816T152540, start_date=20220816T152549, end_date=20220816T152550
[2022-08-16 15:25:51,067] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-16 15:25:51,484] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
